docs/api-docs/webhooks/webhook-events.mdx

```diff
@@ -335,6 +335,31 @@ Payload objects with the following scopes take the form that follows:
 }
 ```
 
+## Delivery exception hooks
+
+| Name / Scope             | Description |
+|:-------------------------|:------------|
+| store/webhook/deliveryException | This hook fires when client hooks are undeliverable.|
```

Comment:
```suggestion
| store/hook/deliveryException | This hook fires when client hooks are undeliverable.|
```
also please include more info like:
- can be just one per (storeId, clientId)
- destination must be different from all others hooks destinations
- once configured will fire when platform had issues with delivering other (regular) hooks.